main-app/src/main/resources/application.yml

```diff
@@ -5,4 +5,13 @@ server:
 
 spring:
   application:
-    name: main-app
\ No newline at end of file
+    name: main-app
+
+env:
```

Comment:
for group RabbitMQ-specific configurations, better give more readable name like "rabbitmq".